kafka/tools/create_volumes.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ux
+
+model=`juju show-model --format=json| jq -r '.| keys[]'`
+machines=`juju status $model --format=json | jq -r '.machines | to_entries[].value."instance-id"'`
+num=`echo "$machines" | wc -l`
+
+for i in $(seq 1 $((num*2))); do
+	openstack volume create kafka-vol-$i --size 1
```

Comment:
to avoid creating volumes we dont need, could check if kafka-vol-$i exists and has status "available" before creating another one.